config/interpolate_funcs.go

```diff
@@ -337,7 +335,7 @@ func interpolationFuncConcat() ast.Function {
 					}
 
 				default:
-					return nil, fmt.Errorf("concat() does not support %T", arg)
+					return nil, fmt.Errorf("concat() does not support type %T", arg)
```

Comment:
I think we might be able to get HIL to do this for us by changing `VariadicType` to `ast.TypeList` above